src/pages/login/LoginPage.tsx

```diff
@@ -81,9 +81,15 @@ export default function LoginPage() {
           dataTestId={TEST_ID.PASSWORD_FIELD}
           error={error.password}
         />
-        <NavLink to={ROUTES.REGISTER}>
-          <FormHelperText>Not registered? Click here!</FormHelperText>
-        </NavLink>
+        <Stack>
+          <NavLink to={ROUTES.REGISTER}>
+            <FormHelperText>Not registered? Click here!</FormHelperText>
```

Comment:
```suggestion
            <FormHelperText>Not registered?</FormHelperText>
```

I'd say "Click here!" part is redundant and it crows a bit the UI without adding any value